app/models/oral_history_content/date_and_place.rb

```diff
@@ -8,5 +9,25 @@ class DateAndPlace
     attr_json :province, :string
     attr_json :country, :string
 
+    def state_name
+      return nil unless country == 'US'
+      ISO3166::Country.new(country).subdivisions[state]['translations']['en']
+    end
+
+    def province_name
+      return nil unless country == 'CA'
+      ISO3166::Country.new(country).subdivisions[province]['translations']['en']
+    end
```

Comment:
Is this really the way the `countries` or `country_select` gem, or wherever this comes from, wants you to go from the code to a human name? I am surprised it's not more straightforward than this. 

Do you know what gem he province and state names are coming from? Can you link to the documentation that suggests using it's API like this? I'm just wondering if there's a different way it's recommending.

Reply:
I got rid of this-- we're just using a simple file with text arrays right now.